components/embl-notifications/embl-notifications.js

```diff
@@ -59,21 +69,18 @@ function emblNotificationsInject(message) {
     }
 
   } else if (message.field_notification_position == 'top') {
-    message.body = message.body.replace(/<[/]?[p>]+>/g, ''); // no <p> tags allowed in inline messages
     output.classList.add('vf-banner', 'vf-banner--fixed', 'vf-banner--top', 'vf-banner--phase');
     output.dataset.vfJsBanner = true;
     output.dataset.vfJsBannerState = message.field_notification_presentation;
-    output.dataset.vfJsBannerButtonText = "Close notice";
+    output.dataset.vfJsBannerButtonText = message.buttonText;
     // These features are not yet supported by the notification content type in the EMBL contentHub
     // output.dataset.vfJsBannerCookieName = "CookieName";
     // output.dataset.vfJsBannerCookieVersion = "CookieVersion";
     // output.dataset.vfJsBannerExtraButton = "<a href='#'>Optional button</a><a target='_blank' href='#'>New tab button</a>";
     // output.dataset.vfJsBannerAutoAccept = false;
-
-    // <h4 class="vf-text vf-text-heading--4"><a class="vf-link" href="${message.field_notification_link}">${message.title}</a></h4>
     output.innerHTML = `
       <div class="vf-banner__content" data-vf-js-banner-text>
-        <p class="vf-text vf-text-body--3">${message.body} <a class="vf-link" href="${message.field_notification_link}">Learn more</a></p>
+        <p class="vf-text vf-text-body--3">${message.body}</p>
```

Comment:
I wonder if this should be `vf-banner__text`. I know typography is now 'calmed'. Ir even `vf-banner__text | vf-text -vf-text-body--3` ?

Happy either way really

Reply:
Good call. We have the vf-banner__text class but weren't using it anywhere. Have added it in for the notifications and banner.